presto-spark-base/src/main/java/com/facebook/presto/spark/execution/NativeExecutionProcess.java

```diff
@@ -177,6 +160,11 @@ public void close()
         }
     }
 
+    public int getPort()
```

Comment:
Can we add some check here so that if port is not yet generated we know we are calling it in a wrong way?

Reply:
It seems that I cannot add comments to places where there are no changes. But below method 
getAvailableTcpPort() does not seem to be used in this file. Can we double check?

Reply:
`getAvailableTcpPort ` is used at L97  ` this.port = getAvailableTcpPort();` That said, since the `port` is initialized in the constructor, the `getPort()` method should be safe to call anytime.